src/components/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes, Component } from 'react'
+import SelectableTimeSlice from '../containers/SelectableTimeSlice.jsx'
+import date from '../utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
+    step: PropTypes.number.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showlabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timeGutterFormat: PropTypes.string,
+    timesliceClassnames: PropTypes.string,
+    culture: PropTypes.string
+  }
+  static defaultProps = {
+    slices: 2,
+    step: 10,
+    isNow: false,
+    showlabels: true,
+    timesliceClassnames: '',
+    culture: 'en'
+  }
+
+  renderSlice(i, classNames, value) {
+    return <SelectableTimeSlice key={i}
+              classNames={classNames}
+              showlabel={this.props.showlabels && !i}
+              format={this.props.timeGutterFormat}
+              culture={this.props.culture}
+              isNow={this.props.isNow}
+              value={value} />
+  }
+
+  renderSlices() {
+    let classNames = this.props.timesliceClassnames
+    switch (this.props.slices) {
```

Comment:
this is (was) used to pass a day-slot specific classname in.  In this commit: https://github.com/intljusticemission/react-big-calendar/pull/102/commits/daae7c84b6869022accd4b8cb6cd5c889f3def7b#diff-6c8b49f3b9d007e9ae15ba43ecc8029aL39 I removed the css in question.  Basically, for many slices, you get lines underneath the text, I thought it looked unclean, but we can talk about that later.